steam_wrapper/steam_wrapper.py

```diff
@@ -204,8 +222,7 @@ def __init__(self, source: str, target: str,
         assert os.path.isabs(self.target)
         self.ignore = ignore or set()
         self.rename = rename or []
-        self.no_copy = self.ignore | set(self.rename)
-        assert not any(os.path.isabs(i) for i in self.no_copy)
+        self.no_copy = FileIgnorer(self.ignore | set(self.rename))
```

Comment:
Nitpick: a `self.ignorer` attribute name would be more sensible.

Reply:
Let's just name it file_ignorer then. There is already an attribute called ignore and potential for typos is significant.